src/CoinbaseWalletSDK.ts

```diff
@@ -167,6 +175,13 @@ export class CoinbaseWalletSDK {
     }
   }
 
+  /**
+   * Return QR URL for mobile wallet connection, will return null if extension is installed
+   */
+  public getQrUrl(): string | null {
+    return this._qrUrl;
```

Comment:
Will this ever return `null`? or just an empty string? .. checking against `createQrUrl`

Reply:
so it's default initiated to null, and when we detect the wallet extension (lines 79-81), it'll not get set and remain null because we won't have created a relay session.
![Screen Shot 2022-04-12 at 12 06 21 PM](https://user-images.githubusercontent.com/15843286/163005921-953f5a4c-b7b8-4e23-a1ba-0f5b41e85867.png)